Editor/PlayerPrefsEditor.cs

```diff
@@ -303,9 +316,17 @@ private PlayerPrefPair[] RetrieveSavedPrefs(string companyName, string productNa
 
             if (showEditorPrefs)
             {
-                string majorVersion = Application.unityVersion.Split('.')[0];
+                // Starting Unity 5.5 registry key has " 5.x" suffix: https://docs.unity3d.com/550/Documentation/ScriptReference/EditorPrefs.html
+                // Even though for some versions of Unity docs state that N.x suffix is used where N.x is the major version number,
+                // it's still " 5.x" suffix used for that cases which is probably bug in the docs.
+                // Note that starting 2019.2 docs have " 5.x" suffix: https://docs.unity3d.com/2019.2/Documentation/ScriptReference/EditorPrefs.html
```

Comment:
Amusingly, even though they've corrected the version number the registry key is still incorrect as it's documented as "UnityEditor 5.x" instead of "Unity Editor 5.x"

Reply:
Good catch! I've probably missed that space as otherwise it should be mentioned in the code comment.

I've reported a problem via Unity docs link on the page for https://docs.unity3d.com/2019.3/Documentation/ScriptReference/EditorPrefs.html about missing space. I remember also reported a problem about N.x usage in https://docs.unity3d.com/2018.4/Documentation/ScriptReference/EditorPrefs.html but documentation is still not fixed unfortunatelly.